api/src/api/language/image.js

```diff
@@ -0,0 +1,56 @@
+const express = require('express');
+const router = express.Router();
+const { errorWrap } = require('../../middleware');
+const { sendResponse } = require('../../utils/response');
+const { models } = require('../../models/index.js');
+const { requireAuthentication } = require('../../middleware/authentication');
+const { uploadFile } = require('../../utils/aws/s3');
+const { ERR_MISSING_OR_INVALID_DATA } = require('../../utils/constants');
+const { checkIds } = require('../../utils/languageHelper');
+
+router.post(
+  '/',
+  requireAuthentication,
+  errorWrap(async (req, res) => {
+    const { lesson_id, vocab_id, course_id, unit_id } = req.body;
+
+    if (!lesson_id || !vocab_id || !course_id || !unit_id) {
+      return sendResponse(res, 400, ERR_MISSING_OR_INVALID_DATA);
+    }
+    const isValid = await checkIds({ lesson_id, course_id, unit_id });
+
+    if (!isValid) {
+      return sendResponse(res, 400, ERR_MISSING_OR_INVALID_DATA);
+    }
+
+    let lesson = await models.Lesson.findOne({ _id: lesson_id }); // find a lesson
```

Comment:
Can you remove this check and update `checkIds` with Jessica's code on the branch `jessica/get-endpoint-language-audio`? Then, pass `vocab_id` into `checkIds` like `const isValid = await checkIds({ lesson_id, course_id, unit_id });`

Finally, can you make the same change to the POST audio endpoint?